src/frontend/src/components/SearchPage/SearchPage.jsx

```diff
@@ -26,10 +26,15 @@ const useStyles = makeStyles(() => ({
 const SearchPage = () => {
   const { telescopeUrl } = useSiteMetadata();
   const classes = useStyles();
-
-  const [searchText, setSearchText] = useState('');
+  let [keyword] = useQueryParam('q', StringParam);
+  let [filters] = useQueryParam('f', StringParam);
+  if (!keyword) {
+    keyword = '';
+    filters = 'post';
+  }
+  const [searchText, setSearchText] = useState(`${keyword}`);
```

Comment:
Let's not mix the query param hook and `useState`.  We shouldn't need to use state at all, just the values coming from `userQueryParam`

Reply:
Remove these